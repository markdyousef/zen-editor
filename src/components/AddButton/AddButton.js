import React, { Component, PropTypes } from 'react';
import styled from 'styled-components';

const Container = styled.div`
    height: 200px;
    width: 100px;
`;

export default class AddButton extends Component {
    static propTypes = {}
    constructor() {
        super();
        this.state = {};
    }
    render() {
        return (
            <Container>
                Yeah
            </Container>
        );
    }
}
