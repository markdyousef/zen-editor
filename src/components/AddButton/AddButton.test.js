import React from 'react';
import renderer from 'react-test-renderer';
import AddButton from './AddButton';

it('renders component correctly', () => {
    const component = renderer.create(<AddButton />);
    expect(component).toMatchSnapshot(component);
});
