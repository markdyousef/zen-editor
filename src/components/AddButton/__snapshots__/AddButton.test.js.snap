exports[`[object Object] 1`] = `
<div
  className="kqVIfr">
  Yeah
</div>
`;
